src/app/showcase/empty-state-showcase/empty-state-showcase.component.ts

```diff
@@ -27,7 +27,7 @@ export class EmptyStateShowcaseComponent {
     },
     {
       name: 'subtitle',
-      description: 'The text beneath the title.',
+      description: "The text beneath the title. Use '\\n' for newline",
```

Comment:
Isn't it only 1 backslash?
```suggestion
      description: "The text beneath the title. Use '\n' for newline",
```

Reply:
the first one escapes the second one :-)